test/widgets/compose_box_test.dart

```diff
@@ -40,14 +42,15 @@ void main() {
     connection = store.connection as FakeApiConnection;
 
     final controllerKey = GlobalKey<ComposeBoxController>();
-    await tester.pumpWidget(
+    await tester.pumpWidget(Builder(builder: (context) =>
       MaterialApp(
+        theme: zulipThemeData(context),
         localizationsDelegates: ZulipLocalizations.localizationsDelegates,
         supportedLocales: ZulipLocalizations.supportedLocales,
         home: GlobalStoreWidget(
           child: PerAccountStoreWidget(
             accountId: eg.selfAccount.id,
-            child: ComposeBox(controllerKey: controllerKey, narrow: narrow)))));
+            child: ComposeBox(controllerKey: controllerKey, narrow: narrow))))));
```

Comment:
To resolve the conflict with the just-merged #805, you should be able to just drop these changes. `TestZulipApp` (used here in #805) takes care of `theme: zulipThemeData(context)` for you.